src/test/java/org/littil/api/guestTeacher/service/GuestTeacherServiceTest.java

```diff
@@ -147,32 +150,27 @@ void givenSaveTeacherUnknownErrorOccurred_thenShouldThrowPersistenceException()
         final UUID userId = UUID.randomUUID();
         final String surname = RandomStringUtils.randomAlphabetic(10);
         final String firstName = RandomStringUtils.randomAlphabetic(10);
-        final String emailAddress = RandomStringUtils.randomAlphabetic(10).concat("@littil.org");
+        final String address = RandomStringUtils.randomAlphabetic(10);
         final String locale = RandomStringUtils.randomAlphabetic(2);
         final String postalCode = RandomStringUtils.randomAlphabetic(6);
 
-        final GuestTeacher guestTeacher = new GuestTeacher();
-        guestTeacher.setSurname(surname);
-        guestTeacher.setFirstName(firstName);
-        guestTeacher.setLocale(locale);
-        guestTeacher.setPostalCode(postalCode);
-
-        final GuestTeacherEntity entity = new GuestTeacherEntity();
-        entity.setId(teacherId);
-        entity.setSurname(surname);
-        entity.setFirstName(firstName);
+        final GuestTeacher guestTeacher = createGuestTeacher(null, firstName, surname, address, postalCode, locale);
+        final GuestTeacherEntity entity = createGuestTeacherEntity(teacherId, firstName, surname);
 
         final GuestTeacher expectedGuestTeacher = new GuestTeacher();
         expectedGuestTeacher.setId(entity.getId());
         expectedGuestTeacher.setSurname(entity.getSurname());
         expectedGuestTeacher.setFirstName(entity.getFirstName());
```

Comment:
ExpectedQuestTeacher is not used, so these lines can be removed.

Reply:
Thanks, this has been removed now.